modules/python/python-tf.c

```diff
@@ -114,6 +120,8 @@ static gboolean
 tf_python_prepare(LogTemplateFunction *self, gpointer s, LogTemplate *parent, gint argc, gchar *argv[],
                   GError **error)
 {
+  PythonTfState *state = (PythonTfState *) s;
+  state->config_version_3_x_compat = cfg_is_config_version_older(parent->cfg, VERSION_VALUE_4_0);
```

Comment:
again the typing specific version check should be used here.

Reply:
Thanks, fixed.